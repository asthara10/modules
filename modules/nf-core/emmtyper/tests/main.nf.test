nextflow_process {

    name "Test Process EMMTYPER"
    script "../main.nf"
    process "EMMTYPER"

    tag "modules"
    tag "modules_nfcore"
    tag "emmtyper"

    test("test_emmtyper") {


        when {
            process {
                """
                
                input[0] = [ [ id:'test', single_end:false ], // meta map
              file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true) ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
