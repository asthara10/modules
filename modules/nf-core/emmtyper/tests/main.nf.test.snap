{
    "versions": {
        "content": [
            [
                "versions.yml:md5,8028be40b22a6bec2ce48bbc811c663a"
            ]
        ],
        "timestamp": "2023-11-24T14:36:14.586303"
    }
}